src/checkpatch_wrapper.sh

```diff
@@ -10,17 +12,41 @@ function execute_checkpatch()
   # --codespell $external_script_path/spelling.txt
 
   local -r options="--terse --no-tree --color=always -strict --file "
-  local -r checkpatch="perl $external_script_path/checkpatch.pl $options"
+  local -r script="scripts/checkpatch.pl $options"
+  local kernel_root=""
```

Comment:
Nitpick: Change this line with the below line; with this, we are going to have a Christmas three order.